Index: 20201101_AnalysisRealEstate_2.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import pandas as pd\nimport numpy as np\nimport matplotlib.pyplot as plt\n\n# 分析\ndf = pd.read_csv(\"sumoAnalyze.csv\",  encoding=\"utf-16\")\ndf.drop(['Unnamed: 0'], axis=1, inplace=True)\nsummary = df.describe()\n\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/20201101_AnalysisRealEstate_2.py b/20201101_AnalysisRealEstate_2.py
--- a/20201101_AnalysisRealEstate_2.py	(revision f5d487abc8ac833161883d91e6b6ce230eadd579)
+++ b/20201101_AnalysisRealEstate_2.py	(date 1611413793891)
@@ -1,6 +1,7 @@
 import pandas as pd
 import numpy as np
 import matplotlib.pyplot as plt
+plt.rc('font', family='Noto Sans CJK JP')
 
 # 分析
 df = pd.read_csv("sumoAnalyze.csv",  encoding="utf-16")
